packages/headless-react/typedoc/configs/ssr-commerce.typedoc.json

```diff
@@ -0,0 +1,24 @@
+{
+  "navigation": {
+    "includeCategories": true,
+    "includeGroups": true,
+    "includeFolders": true
+  },
+  "categorizeByGroup": true,
+  "groupOrder": [
+    "React",
+    "Engine",
+    "Definers",
+    "Controllers",
+    "Buildable controllers",
+    "Sub-controllers",
```

Comment:
In the case of SSR this distinction becomes confusing. Could we group them all under "Controllers"?

Reply:
Also I notice that some controllers actually appear in multiple sections, making this even more confusing.

@jpmarceau perhaps we should just get rid of this distinction altogether... thoughts?

Reply:
"perhaps we should just get rid of this distinction altogether... thoughts?"
Interesting idea
It's true that with SSR, you'll use hooks anyway, and you'll never need the concept of "sub"-controller. So yeah, seems to me like it would be simpler to get rid the the distinction within the scope of SSR 👍

Reply:
The exported artifacts of this package is 100% of headless/ssr-commerce + some React functions. This is why I readded those groups. the point is that this is an exact copy of the docs of headless/ssr-commerce   + a new group for the React specific stuff.